db/migrate/20200129172534_add_epoch_version_release_arch_to_katello_installed_packages.rb

```diff
@@ -0,0 +1,35 @@
+class AddEpochVersionReleaseArchToKatelloInstalledPackages < ActiveRecord::Migration[5.2]
+  def up
+    add_column :katello_installed_packages, :nvrea, :string
+    add_column :katello_installed_packages, :epoch, :string
+    add_column :katello_installed_packages, :version, :string
+    add_column :katello_installed_packages, :release, :string
+    add_column :katello_installed_packages, :arch, :string
+
+    epoch_non_0 = ::Katello::Rpm.where.not(epoch: [0, nil]).pluck(:nvra, :epoch)
+
+    ::Katello::InstalledPackage.reset_column_information
+    installed_packages = ::Katello::InstalledPackage.all.collect do |pkg|
```

Comment:
I'd probably use InstalledPackage.find_each for this: https://api.rubyonrails.org/classes/ActiveRecord/Batches.html#method-i-find_each